### PR TITLE
SSCS TYA demo and AAT redirect

### PR DESCRIPTION
**Pull request by rahman124**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: environments/stg/stg.tfvars [hunk 1]
```diff
@@ -207,6 +207,13 @@ frontends = [
     backend_domain   = ["firewall-prod-int-palo-aat.uksouth.cloudapp.azure.com"]
     certificate_name = "wildcard-aat-platform-hmcts-net"
   },
+  {
+    name             = "sscs-tya-redirect"
+    mode             = "Detection"
+    custom_domain    = "sscs-cor.aat.platform.hmcts.net/validate-surname/:tya/trackyourappeal"
```
Comment (naikajah) [inline environments/stg/stg.tfvars:213]: Is this correct? I think this needs to be just a custom domain name without "/"

Comment (adusumillipraveen) [inline environments/stg/stg.tfvars:213]: will this work ? this should be a domain ? not url

Comment (rahman124) [reply environments/stg/stg.tfvars:213]: its for redirecting http://track-appeal.aat.platform.hmcts.net/* 
to 
http://sscs-cor.aat.platform.hmcts.net/validate-surname/:tya/trackyourappeal
should I still change it to sscs-cor.aat.platform.hmcts.net

Comment (naikajah) [reply environments/stg/stg.tfvars:213]: not sure if this will work. As this is supposed to be a domain and not url.

**Pull request merged**